qcodes/instrument_drivers/tektronix/Keithley_6500.py

```diff
@@ -109,10 +109,15 @@ def __init__(self, name, address, reset_device=False, **kwargs):
                            vals=Bool())
 
         # Global parameters
-        self.add_parameter('display_enabled',
-                           get_parser=_parse_output_bool,
-                           get_cmd='DISP:ENAB?',
-                           set_cmd='DISP:ENAB {}', set_parser=int, vals=Bool())
+        self.add_parameter('display_backlight',
+                           docstring='Control the brightness of the display '
+                                     'backligt. OFF turns the display off and'
+                                     'BLACkout also turns off indicators and '
+                                     'key lights on the device.',
+                           get_cmd='DISP:LIGH:STAT?',
+                           set_cmd='DISP:LIGH:STAT {}',
+                           vals=Enum('ON100', 'ON75', 'ON50', 'ON25', 'OFF',
```

Comment:
i'd recommend using val_mapping here and give user-friendly names to each value.

Reply:
Done